db/db_iter.cc

```diff
@@ -667,6 +667,8 @@ inline bool DBIter::FindNextUserEntryInternal(bool skipping, bool prefix_check)
 // POST: saved_value_ has the merged value for the user key
 //       iter_ points to the next entry (or invalid)
 bool DBIter::MergeValuesNewToOld() {
+  // @TODO(tabokie): this function asserts we have value to yield.
```

Comment:
what does this todo mean? Cleanup the comments.

Reply:
(This is the major setback for delete by merge)